app/src/androidTest/java/com/xmartlabs/dd/ui/common/BaseInstrumentationTest.java

```diff
@@ -29,7 +29,7 @@
   @CallSuper
   public void setUp() {
     instrumentation = InstrumentationRegistry.getInstrumentation();
-    BaseProjectApplication app = (BaseProjectApplication) instrumentation.getTargetContext().getApplicationContext();
+    DDApplication app = (DDApplication) instrumentation.getTargetContext().getApplicationContext();
```

Comment:
But DD is a acronym, for example uri is an acronym to and the right name is Uri. Maybe you should use `DayDreaming` in the whole project. Why are you using `DD`?